packages/js/src/utils/scope.test.ts

```diff
@@ -0,0 +1,82 @@
+import { DEFAULT_SCOPE_STRING, ScopeValue, withDefaultScopes } from './scope';
+
+describe('withDefaultScopes', () => {
```

Comment:
Should be better if we use natural words instead of camel case. E.g. `Generate sign-in URI with default scopes`

Reply:
i'm used to naming the top level description with the name of function i test.
i'm ok with both.